server/tests-py/validate.py

```diff
@@ -274,18 +278,32 @@ def validate_http_anyq(hge_ctx, url, query, headers, exp_code, exp_response):
     else:
         return resp, True
 
+def validate_http_anyq_with_allowed_responses(hge_ctx, url, query, headers, exp_code, allowed_responses):
+    code, resp, resp_hdrs = hge_ctx.anyq(url, query, headers)
+    print(headers)
+    assert code == exp_code, resp
+    print('http resp: ', resp)
+    if allowed_responses:
```

Comment:
I'm concerned/confused about the conditional  here. As i understand it, the `else` branch is taken on the empty list? Maybe it would be better to just throw an error here, do you think so?

Reply:
Yes, this makes sense. I will modify this. 👍